service_capacity_modeling/models/org/netflix/cassandra.py

```diff
@@ -215,15 +215,19 @@ def _estimate_cassandra_cluster_zonal(
     require_attached_disks: bool = False,
     required_cluster_size: Optional[int] = None,
     max_rps_to_disk: int = 500,
-    max_local_disk_gib: int = 2048,
-    max_regional_size: int = 96,
+    max_local_disk_gib: int = 5120,
+    max_regional_size: int = 192,
     max_write_buffer_percent: float = 0.25,
     max_table_buffer_percent: float = 0.11,
 ) -> Optional[CapacityPlan]:
     # Netflix Cassandra doesn't like to deploy on really small instances
     if instance.cpu < 2 or instance.ram_gib < 14:
         return None
 
+    # temporarily dont suggest EBS instances
+    if instance.drive is None:
+        return None
+
     # if we're not allowed to use gp2, skip EBS only types
     if instance.drive is None and require_local_disks:
```

Comment:
can you just flip the default of require_local_disks to true?

Reply:
Good point. I have fixed it along with the tests.